src/styles.css

```diff
@@ -1,9 +1,9 @@
-[data-contentful-field-id] {
+[data-contentful-field-id][data-contentful-entry-id][data-contentful-locale] {
```

Comment:
Show only the borders if everything is configured correctly (caused confusion, because the border was shown, but no edit button)